src/XMakeTasks/Copy.cs

```diff
@@ -408,6 +409,20 @@ CopyFileWithState copyFile
                 _copiedFiles = new TaskItem[0];
                 return true;
             }
+            
+            if (_destinationFiles != null)
+            {
+                var sameDestinationFiles = _destinationFiles.GroupBy(x => x).Where(g => g.Count() > 1).Select(y => y.First());
```

Comment:
@rainersigwald, what do you think about that.
```C#
 if (_destinationFiles != null)
            {
                var sameDestinationFiles = _destinationFiles.GroupBy(x => x.ItemSpec).Where(g => g.Count() > 1);
                
                if (sameDestinationFiles.Any())
                {
                    for(int i = 0; i < _sourceFiles.Length; ++i)
                    {
                        // Copy meta-data from source to destinationFiles.
                        _sourceFiles[i].CopyMetadataTo(_destinationFiles[i]);
                    }

                    foreach (var item in sameDestinationFiles)
                    {
                        string logSource = "";
                        foreach(var subitem in item)
                        {
                            logSource += '"' + subitem.GetMetadata("OriginalItemSpec") + "\",";
                        }

                        string logDestination = '"' + item.Key + '"';
                        Log.LogErrorFromResources("Copy.SameDestinationPath", logSource, logDestination);
                    }
                    return false;
                }
            }
```
then error looks like this
![image](https://cloud.githubusercontent.com/assets/19551763/18749252/3d33637e-80d6-11e6-93ec-f1bdca818dfd.png)